doc/man1/flux-fsck.rst

```diff
@@ -40,6 +40,11 @@ OPTIONS
    checkpoint.  This option directs flux-fsck to start at an arbitrary
    point.  BLOBREF must refer to an RFC 11 tree object of type "dir".
 
+.. option:: -l, --lost-and-found
+
+   Repair corrupted KVS entries and copy them into the `lost+found` directory.
+   Will update the primary KVS checkpoint after `fsck` is complete.
+
```

Comment:
This should probably be expanded a bit e.g.

> Remove any dangling references found in KVS metadata.  If a KVS value changes as a result of this repair, the key is moved to the `lost+found` directory.  If a key points to a KVS directory that is no longer valid, it is unlinked.  All damaged keys and their disposition are listed on stderr.  This process creates a new root reference for these changes, and commits it as the current KVS checkpoint at the end of the scan.  The KVS is required to be unloaded during repair.

Reply:
i put some thoughts about dirref repair into #6589.  Minimally, lets punt on it to a follow on PR.

Reply:
I just noticed a subtlety in your description.  You say "The key is **moved**" , do you think we should unlink the original?  i.e. if we repair `dir.a.b`, should we remove `dir.a.b` and add `lost+found.dir.a.b`?